t/sanity.t

```diff
@@ -863,6 +863,7 @@ GET /t
 dog not found
 --- no_error_log
 [error]
+--- only
```

Comment:
todo: I forgot to remove it. This bug is not exposed because lower case doesn't take effect.